fastplotlib/widgets/image.py

```diff
@@ -79,9 +79,9 @@ def __repr__(self):
 
 class ImageWidget:
     @property
-    def plot(self) -> Union[Plot, GridPlot]:
+    def plot(self) -> GridPlot:
```

Comment:
let's just rename this to `gridplot` and then I will have fun refactoring elsewhere in mesmerize etc. :smile: